v8env_test/request.spec.js

```diff
@@ -1,14 +1,87 @@
 import { expect } from 'chai'
 
 describe("Request", () => {
+  const text = "helloFLY"
+  const buffer = new TextEncoder("utf-8").encode(text).buffer
+  const buffer16 = new TextEncoder("utf-16").encode(text).buffer
+  const buffer32 = new TextEncoder("utf-32").encode(text).buffer
+  const buffer64 = new TextEncoder("utf-64").encode(text).buffer
```

Comment:
I don't believe those different `utf-x` encodings will work the way you're expecting them to.

`TextEncoder` takes no arguments and it's `encode` function will always return a Uint8Array. I think the spec says it only accepts utf-8 encoded string too.

I think you might be able to use `TypedArray.from` to convert between different types. I'm not entirely sure how it will work though. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray/from

I'm not sure what the result from those will be though! You may need to find better sources of bytes for those different types of TypedArrays.